admin/css/debug-toolbar/toolbar.scss

```diff
@@ -467,6 +467,17 @@
     @import '_theme-light';
 }
 
+// show tab
+#ci-database.debug-bar-dblock,
+#ci-files.debug-bar-dblock,
+#ci-routes.debug-bar-dblock,
+#ci-events.debug-bar-dblock,
+#ci-history.debug-bar-dblock,
+#ci-vars.debug-bar-dblock,
+#ci-config.debug-bar-dblock,
+#ci-timeline.debug-bar-dblock {
```

Comment:
I believe this can be simplified to one CSS class, `.debug-bar-dblock`, instead of individually showing all uses in `#ci-*` ids. If there would be a new id, say `#ci-dump`, it would need to be added here again.

Reply:
You're right..thanks @paulbalandan